src/registry/api.py

```diff
@@ -14,7 +14,7 @@
 from main.schema import ForbiddenSchema, NotFoundSchema, Schema, SuccessSchema
 from users.auth import UidKeyAuth
 
-from .models import Author, Model, Prediction
+from .models import Author, Model, Prediction, ImplementationLanguage
```

Comment:
I am thinking that maybe this model doesn't need to me exposed through the API it is just for internal use and we will update the table via the Django-admin

Reply:
I had to add the Implementation Language check on [create_model](https://github.com/Mosqlimate-project/Data-platform/pull/62/files/cfb3de673b7666407d8b4464c86ab196df734ee0#diff-bb6dc47149a579fd6505226a53dfab76f802304d1841703c2f98f1a509a7facdL177) API call because the call will only accept languages that have already been added as an `ImplementationLanguage` object. If the user tries to specify an unknown Implementation Language, it will return an 404 error: https://github.com/Mosqlimate-project/Data-platform/pull/62/files/cfb3de673b7666407d8b4464c86ab196df734ee0#diff-bb6dc47149a579fd6505226a53dfab76f802304d1841703c2f98f1a509a7facdR203

Reply:
ok